src/Sentry/SentryLogger.cs

```diff
@@ -0,0 +1,68 @@
+using Sentry.Experimental;
+using Sentry.Infrastructure;
+using Sentry.Protocol.Envelopes;
+
+//TODO: add XML docs
+#pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
+
+namespace Sentry;
+
+/// <summary>
+/// Creates and sends logs to Sentry.
+/// </summary>
+[Experimental(DiagnosticId.ExperimentalSentryLogs, UrlFormat = UrlFormats.ExperimentalSentryLogs)]
+public sealed class SentryLogger
+{
+    //TODO: QUESTION: Trace vs LogTrace
+    // Trace() is from the Sentry Logs feature specs. LogTrace() would be more .NET idiomatic
+    public void Trace(string template, object[]? parameters = null, Action<SentryLog>? configureLog = null)
+    {
+        CaptureLog(SentrySeverity.Trace, template, parameters, configureLog);
+    }
+
+    //TODO: QUESTION: parameter name "template" vs "format"
```

Comment:
I feel like it's less mental gymnastics for us if we go with the sentry naming conventions (when we're reading code from other eepos etc. to see how they make changes that we need to mimic). So I'd probably go with `template` personally.